apps/src/templates/sectionProgress/standards/sectionStandardsProgressRedux.js

```diff
@@ -305,19 +305,21 @@ export function getPluggedLessonCompletionStatus(state, lesson) {
     const numberStudentsInSection =
       state.teacherSections.sections[state.teacherSections.selectedSectionId]
         .studentCount;
-    const levelResultsByStudent =
+    const levelProgressByScript =
       state.sectionProgress.studentLevelProgressByScript[scriptId];
 
-    const studentIds = Object.keys(levelResultsByStudent);
-    const levelIds = _.map(lesson.levels, 'activeId');
+    const studentIds = Object.keys(levelProgressByScript);
+    const levelIds = _.map(lesson.levels, 'id');
     let numStudentsCompletedLesson = 0;
     let numStudentsInProgressLesson = 0;
     studentIds.forEach(studentId => {
       let numLevelsInLessonCompletedByStudent = 0;
       levelIds.forEach(levelId => {
+        const levelProgress = levelProgressByScript[studentId][levelId];
+        // console.log('progress', studentId, levelId, levelProgress);
```

Comment:
👀  rogue log

Reply:
thanks! as you can tell, i had some trouble fixing tests for this module ;-)